app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+
+    if len(split_command) == 3:
+        comm, file_in, file_path = split_command
+
+        if comm == "mv":
+            path, _ = os.path.split(file_path)
+            if not path:
+                path = "./"
+            if file_path.endswith("/"):
+                path = file_path
+            elif len(path) > 1 and not os.path.exists(path):
+                os.makedirs(path, exist_ok=True)
```

Comment:
So hard logic as for me. Simplify it and just do:
```suggestion
            if path:
                 os.makedirs(path, exist_ok=True)
```